.github/workflows/check-versions.yaml

```diff
@@ -0,0 +1,61 @@
+name: Check versions
+
+on: pull_request
```

Comment:
Per [this action's docs](https://github.com/marketplace/actions/comment-pull-request#permissions), I anticipate that this will change to `pull_request_target` so that forked PRs have the necessary permissions for GitHub to add a comment.